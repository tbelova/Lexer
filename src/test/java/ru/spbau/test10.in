
while (i < 1) {
	x := 5
}
