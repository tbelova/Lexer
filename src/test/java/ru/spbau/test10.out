line 2:14
no viable alternative at input 'while(i<1){'